### ISSUE TITLE
Model and Label file paths should not be hard coded

### ISSUE THREAD

**Issue by robmarkcole** [robmarkcole/coral-pi-rest-server#1]
As title

**Comment by robmarkcole** [comment]
Add a config.json file

**Comment by robmarkcole** [comment]
Furthermore do a check on server start that the file paths are valid

**Comment by lmamakos** [comment]
It's too late in the evening to worry about merging your changes into my cloned repo; I just smashed in some changes to your recent release to add in options for the model and labels file.  See here...

[patch.txt](https://github.com/robmarkcole/coral-pi-rest-server/files/3459736/patch.txt)

I also added back in the `/predict` URI path as a transition feature.

I didn't manage to merge your changes into my repo, and I think attempting to do that now will delay things further as I'm going to be out of pocket for a few days..

**Comment by robmarkcole** [comment]
Any thoughts on whether its preferable to:

1. Pass in environment variables
2. Mount a dir with a config file
3. Use args as your example

My thought is whatever results in a nice docker-compose is preferable, but I'm not too experienced with docker to make these calls yet.

**Comment by lmamakos** [comment]
I'm very far from a Docker expert, so my judgement of best practices is suspect.

I'm thinking that primarily the variability here is what model someone might want to use, and you pretty much have to mount a volume in the container if you want to use a single prebuilt container with different models.  It might make sense to use an environment variable if you wanted to control logging verbosity or something, but I expect that would be pretty rare.

You could actually pass in a "volume" which is a single file,  if you wanted to.  So maybe have the model in the container live in `/config/model.tflite` and the labels as `/config/labels.txt`   With that, you could just externally make a directory with `model.tflite` and `labels.txt` files and mount it on `/config/` or mount to arbitrary files on the .tflite and labels.txt file.

The scheme I used could let you do a similar thing with environment variables and, but I just passed those in as options.  If you do use environment variables, you're still stuck with mounting a volume from the host to make the files accessible, so this is mostly a question of when the binding of the name happens between the outside world and container.

**Comment by robmarkcole** [comment]
I guess the simplest approach would be to mount a volume that contains a `.tfilte` and `.txt` file and these are then used

**Comment by robmarkcole** [comment]
If we use the disk images from https://github.com/google-coral/edgetpu-platforms#prebuilt-images-for-raspberry-pis then we can set a default model, and just allow as a command line arg to select a different model

**Comment by lmamakos** [comment]
In earlier pull request, I added some command line options to allow you to select the model and labels file; you might see about pulling those back in.  I also tried to do something about logging and verbose/quiet mode.. with less success.  I got stuck trying to convince the Flask library to change logging levels and didn't really succeed.

Related, I re-worked the Dockerfile to make it easier to mount a different directory in the container over the model and labels file.  Or just to mount each individual file over the existing one. With the present version of `coral-app.py` I ended up making those hardcoded paths inside the contain be symlinks to more generically named files to make it easier to replace them from outside the container when it's started.

[Dockerfile.txt](https://github.com/robmarkcole/coral-pi-rest-server/files/3484375/Dockerfile.txt)

(Sorry about `.txt` extension; github insists...)

It would be interesting if someone could try building this container on a Raspberry Pi and see if it works.  This version is also much smaller than the previous container build, as it uses a much more stripped down python-3.6 container as a starting point, about 380MB vs 1.1GB previously.
